nautobot/extras/context_managers.py

```diff
@@ -152,3 +159,57 @@ def web_request_context(
         for object_change in ObjectChange.objects.filter(request_id=change_context.change_id).iterator():
             enqueue_job_hooks(object_change)
             enqueue_webhooks(object_change)
+
+
+@contextmanager
+def object_changelogs_bulk_operation(objs, user, action):
```

Comment:
Please add a detailed docstring to this function. I'm also wondering if we should expose this in `nautobot.apps` so that Apps can take advantage of it?